firmware/quadruna/VC/src/app/states/app_initState.c

```diff
@@ -12,6 +23,14 @@ static void initStateRunOnTick1Hz(void)
 static void initStateRunOnTick100Hz(void)
 {
     app_allStates_runOnTick100Hz();
+
+    bool is_inverter_on_or_drive_state =
+        app_canRx_BMS_State_get() == BMS_INVERTER_ON_STATE || app_canRx_BMS_State_get() == BMS_DRIVE_STATE;
```

Comment:
Need to check if BMS is also in precharge `app_canRx_BMS_State_get() == BMS_PRECHARGE_STATE`

Reply:
fixed!